src/webui/webapplication.cpp

```diff
@@ -68,12 +67,12 @@
 
 constexpr int MAX_ALLOWED_FILESIZE = 10 * 1024 * 1024;
 
-const QString PATH_PREFIX_IMAGES {QStringLiteral("/images/")};
```

Comment:
These are legit uses of `QStringLiteral`, why remove it?

Reply:
Sorry, it was rebase artifact. Will be fixed.